jbpm-console-ng-executor-service/jbpm-console-ng-executor-service-backend/src/main/java/org/jbpm/console/ng/es/backend/server/RequestSummaryHelper.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2012 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.console.ng.es.backend.server;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.jbpm.console.ng.es.model.ErrorSummary;
+import org.jbpm.console.ng.es.model.RequestParameterSummary;
+import org.jbpm.console.ng.es.model.RequestSummary;
+import org.kie.api.executor.ErrorInfo;
+import org.kie.api.executor.RequestInfo;
+import org.kie.api.executor.STATUS;
+import org.kie.server.api.model.instance.ErrorInfoInstance;
+import org.kie.server.api.model.instance.RequestInfoInstance;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RequestSummaryHelper {
```

Comment:
This class is probably not needed any more as all the mapping is done via *Mapper classes now.